packages/blade/src/components/List/__tests__/List.web.test.tsx

```diff
@@ -195,6 +233,17 @@ describe('<List />', () => {
     expect(container).toMatchSnapshot();
   });
 
+  it('should render large ordered-filled List', () => {
```

Comment:
let's add a test for list item with <Code\> component inside?

Reply:
There is one such test already